modules/mono/managed_callable.cpp

```diff
@@ -85,6 +85,11 @@ ObjectID ManagedCallable::get_object() const {
 	return CSharpLanguage::get_singleton()->get_managed_callable_middleman()->get_instance_id();
 }
 
+int ManagedCallable::get_argument_count() const {
+	// TODO: I'm not sure if this is needed as I don't know where this is used
+	return 0;
```

Comment:
This would be called when you call `get_argument_count` on a `Callable` created from a C# delegate/lambda. You'd need to create a method in `DelegateUtils.cs` like the `DelegateEquals` and `DelegateHash` methods because we can only retrieve the delegate information from C#.

Reply:
Good, then I will see how that can work out, should be workable

Reply:
I've solved this one, and discovered that the signal awaiters are accessible from the GDScript side with `get_connections` on a `Signal` so trying out some code, will push soon